src/tiles/list/tile-list.ts

```diff
@@ -0,0 +1,374 @@
+/* eslint-disable lit/no-invalid-html */
+import { css, html, LitElement, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { join } from 'lit/directives/join.js';
+import { map } from 'lit/directives/map.js';
+import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import DOMPurify from 'dompurify';
+import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
+
+@customElement('tile-list')
+export class TileList extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
+  render() {
+    return html`
+      <div id="list-line" class="${this.classSize}">
+        <div id="list-line-left">
+          <div id="thumb">${this.imgTemplate}</div>
+          ${this.iconLeftTemplate}
+        </div>
+        <div id="list-line-right">
+          <div id="title">${this.titleTemplate}</div>
+          ${this.itemLineTemplate} ${this.creatorTemplate}
+          <div id="date" class="metadata">
+            <span class="label">Published:</span> ${formatDate(
+              this.date,
+              this.formatSize
+            )}
+          </div>
+          <div id="views-line">
+            ${this.viewsTemplate} ${this.ratingTemplate} ${this.reviewsTemplate}
+          </div>
+          ${this.topicsTemplate} ${this.collectionsTemplate}
+          ${this.descriptionTemplate}
+        </div>
+      </div>
+    `;
+  }
+
+  // Display templates
+  private get imgTemplate() {
+    if (!this.model?.identifier) {
+      return nothing;
+    }
+    return html` <img
+      src="${this.baseNavigationUrl}/services/img/${this.model.identifier}"
+      alt="${this.model.identifier}"
+      class=${this.model?.mediatype}
+    />`;
+  }
+
+  private get iconLeftTemplate() {
+    if (this.classSize !== 'desktop') {
+      return nothing;
+    }
+    return html`
+      <div id="icon-left">
+        <mediatype-icon
+          .mediatype=${this.model?.mediatype}
+          .showText=${true}
+          style="--iconHeight: 20px; --iconWidth: 20px;text-align: center;"
```

Comment:
Could you move these inlines to the `styles()` section?

Reply:
done